config.yaml.sample

```diff
@@ -188,9 +188,16 @@ toi-store:
 
 # Configuration for the tiles of interest prune/garden command.
 toi-prune:
+  # location of tileserver logs
+  tile-traffic-log-path: ../tileserver/nohup.out
+
   # Connection and query configuration for a RedShift database containing
   # request information for tiles.
   redshift:
+    # if database-uri is not specified, postgresql configuration above is used
```

Comment:
@iandees, @rmarianski: so I ditched the fallback to default postgresql configuration in 2ca05fe
generic postgres URI like `postgresql://localhost:5432/osm` works as excepted.

Also I renamed `redshift` entry to `tile-history` for clarity. What do you think?